tests/cpp/BUILD

```diff
@@ -110,21 +89,6 @@ cc_test(
     ],
 )
 
-cc_test(
```

Comment:
Any particular reason, we are getting rid of this?

Reply:
I moved this test to python to compare with pytorch nn.Module directly using cosine sim metric